quasar-utils/javac-plugin/build.gradle

```diff
@@ -0,0 +1,47 @@
+plugins {
+    id 'java-library'
+    id 'maven-publish'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junit_jupiter_version
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: junit_jupiter_version
+    testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit_jupiter_version
+
+    if(JavaVersion.current() <= JavaVersion.VERSION_1_8) {
```

Comment:
```groovy
if (JavaVersion.current().java8Compatible) {
   ...
}
```

Reply:
this looks to me like a less readable way to express the same thing...

Reply:
Of course it does the same thing. Anything else would be a bug :expressionless:.